### PR TITLE
feat(phone-number): Add tests

### PR DESCRIPTION
**Pull request by umahaea**

### CODE REVIEW

File: src/main/java/elementsSDK/functional/fixtures/phone-number.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Compounds : Phone Number</title>
```
Comment (udhadpa) [inline src/main/java/elementsSDK/functional/fixtures/phone-number.html]: I think we should start changing the title to Elements SDK - <component name>
Line 11 follows this pattern

File: src/test/java/elementsSDKTests/functionalTests/PhoneNumberTest.java [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+        commonUtils.click(phNumPgObj.downIcon, "mobile");
+        By item = By.xpath("//button[@aria-label='" + countries[1] + "']");
+        commonUtils.clickUsingJS(item, "mobile");
+        commonUtils.clickUsingJS(item, "mobile");
+        width = commonUtils.getCSSValue(phNumPgObj.flagImg, "width", "mobile");
+        height = commonUtils.getCSSValue(phNumPgObj.flagImg, "height", "mobile");
+
+        //styles between label and flag
+        paddingTop = commonUtils.getCSSValue(phNumPgObj.selectDiv, "padding-top", "mobile");
+        isPaddingTop = commonUtils.assertValue(paddingTop, expPaddingTop, inputType + " padding-top between label and flag as per the spec");
```
Comment (udhadpa) [inline src/test/java/elementsSDKTests/functionalTests/PhoneNumberTest.java]: no assertion for paddingTop

File: src/test/java/elementsSDKTests/functionalTests/PhoneNumberTest.java [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+        //flag styles
+        commonUtils.click(phNumPgObj.downIcon);
+        By item = By.xpath("//button[@aria-label='" + countries[1] + "']");
+        commonUtils.click(item);
+        width = commonUtils.getCSSValue(phNumPgObj.flagImg, "width");
+        height = commonUtils.getCSSValue(phNumPgObj.flagImg, "height");
+
+        //styles between label and flag
+        paddingTop = commonUtils.getCSSValue(phNumPgObj.selectDiv, "padding-top");
+        isPaddingTop = commonUtils.assertValue(paddingTop, expPaddingTop, inputType + " padding-top between label and flag as per the spec");
```
Comment (udhadpa) [inline src/test/java/elementsSDKTests/functionalTests/PhoneNumberTest.java]: Assertion missing for paddingTop

Comment (umahaea) [reply src/test/java/elementsSDKTests/functionalTests/PhoneNumberTest.java]: wow good catch!

File: src/test/java/elementsSDKTests/functionalTests/PhoneNumberTest.java [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+    JavascriptExecutor js = null;
+    PhoneNumberPageObjects phNumPgObj = null;
+    String[] countries = {"India (भारत) +91", "United States +1", "United Kingdom +44", "Spain (España) +34", "France (République française) +33"};
+    String[] countryCodes = {"+91", "+1", "+44", "+34", "+33"};
+    String[] countryFlagCodes = {"in", "us", "gb", "es", "fr"};
+
+    @BeforeClass(alwaysRun = true)
+    private void beforeClass() {
+        commonUtils.readInitialConfig(phoneNumberJSFilePath, tempJSFilePath);
+
```
Comment (udhadpa) [inline src/test/java/elementsSDKTests/functionalTests/PhoneNumberTest.java]: extra space here

**Comment by umahaea** [comment]
all approved..

**Comment by umahaea** [comment]
all approved. merging it.

**Pull request merged**